recipes/libsmacker/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from contextlib import contextmanager
+import os
+
+
```

Comment:
```suggestion
import os

required_conan_version = ">=1.29.1"


```

due to `tools.remove_files_by_mak()`